charts/common/values.yaml

```diff
@@ -299,6 +299,13 @@ components:
       # envConfigMap is a dictionary of environment variables set for every container in a pod, key: value
       # envConfigMap: {}
 
+      # envFrom is a list of environment references
+      envFrom: []
```

Comment:
This would be controllerValues and not containerValues, maybe add both?

Reply:
Added envFrom for container and controller